src/components/ThirdPartyButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { StyleSheet, TouchableOpacity, Image } from 'react-native';
+
+type Props = {
+  imgSrc: string;
+  handleClick: () => void;
+  width: number;
+  height: number;
+};
+
+const ThirdPartyButton: React.FC<Props> = ({ imgSrc, handleClick, width, height }: Props) => {
+  return (
+    <TouchableOpacity onPress={handleClick} style={styles.buttonContainer}>
```

Comment:
Hi @sivaShilpa! :) Can you replace `TouchableOpacity` with `Pressable`? According to the docs it's a more future-proof way to handle the input: https://reactnative.dev/docs/touchableopacity